modules/ossm-remove-control-plane.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/removing-ossm.adoc
+
+[id="ossm-remove-control-plane_{context}"]
+= Removing the control plane
+
+Follow this procedure to remove the {ProductName} control plane.
+
+.Prerequisites
+
+* You must log in as a cluster administrator to run these commands.
+
+.Procedure
+
+[NOTE]
+====
+When you remove the custom resource, {ProductShortName} tells the Operator to begin uninstalling everything it installed.
+====
+
+
+. Run this command to retrieve the name of the installed custom resource:
++
+
+----
+$ oc get servicemeshcontrolplanes -n istio-system
+----
+
++
+. Run this command to remove the custom resource:
++
+----
+$ oc delete -n istio-system -f <name_of_custom_resource>
+----
```

Comment:
The right command is `oc delete -n istio-system <name_of_custom_resource>` (there's no `-f`).
Also, this `<name_of_custom_resource>` is the output of the above command.

Reply:
Fixed.